lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -525,3 +536,144 @@ def delete_bearer(self, imsi, lbi, ebi):
             sid=SIDUtils.to_pb(imsi), link_bearer_id=lbi, eps_bearer_ids=[ebi]
         )
         self._stub.DeleteBearer(req)
+
+
+class SessionManagerUtil(object):
+    """
+    Helper class to communicate with session manager for the tests.
+    """
+
+    def __init__(self):
+        """
+        Initialize sessionManager util.
+        """
+        self._session_stub = SessionProxyResponderStub(
+            get_rpc_channel("sessiond")
+        )
+        self._directorydstub = GatewayDirectoryServiceStub(
+            get_rpc_channel("directoryd")
+        )
+
+    def create_ReAuthRequest(self, imsi, policy_id, flow_list, qos):
+        """
+        Sends Policy RAR message to session manager
+        """
+        print("Sending Policy RAR message to session manager")
+        flow_match_list = []
+        ipv4_src_addr = None
+        ipv4_dst_addr = None
+        res = None
+        for flow in flow_list:
+            if flow["direction"] == "UL":
+                flow_direction = FlowMatch.UPLINK
+            else:
+                flow_direction = FlowMatch.DOWNLINK
+
+            ip_protocol = flow["ip_proto"]
+            if ip_protocol == "TCP":
+                ip_protocol = FlowMatch.IPPROTO_TCP
```

Comment:
how about making a function maybe called `get_flow_match`, just to reduce complexity of the main create_ReAuthRequest function?